prime-router/metadata/fhir_mapping/hl7/datatype/Annotation.yml

```diff
@@ -13,15 +16,15 @@ time:
   valueOf: "GeneralUtils.dateTimeWithZoneId(dateTimeIn,ZONEID)"
   expressionType: JEXL
   vars:
-    dateTimeIn: NTE.6 | NTE.7
```

Comment:
No action necessarily, but just putting this out there: To me, it makes sense to set the FHIR time resource to `Effective Start Date ` if `Entered Date/Time` is not set, even if the mapping inventory does not indicate a mapping for this.

Reply:
I don't feel strongly.  Just would want to call out that I think maybe blows up the scope of this work since a dev now needs to make a qualitative decision for anything not in NIST or mapped if it can be added